code/datums/martial/plasma_fist.dm

```diff
@@ -52,10 +60,30 @@
 	D.visible_message("<span class='danger'>[A] hits [D] with THE PLASMA FIST TECHNIQUE!</span>", \
 					"<span class='userdanger'>You're suddenly hit with THE PLASMA FIST TECHNIQUE by [A]!</span>", "<span class='hear'>You hear a sickening sound of flesh hitting flesh!</span>", null, A)
 	to_chat(A, "<span class='danger'>You hit [D] with THE PLASMA FIST TECHNIQUE!</span>")
+	var/turf/Dturf = get_turf(D)
 	D.gib()
+	if(plasma_power < 12)
+		plasma_power++
+		to_chat(A, "<span class='nicegreen'>Power increasing! Your </span><span class='notice'>Apotheosis</span><span class='nicegreen'> is now at power level [plasma_power]!</span>")
+		new /obj/effect/temp_visual/plasma_soul(Dturf, A)
+		var/oldcolor = A.color
+		A.color = "#9C00FF"
+		flash_color(A, flash_color = "#9C00FF", flash_time = 3 SECONDS)
+		animate(A, color = oldcolor, time = 3 SECONDS)
+	else
+		to_chat(A, "<span class='warning'>You cannot power up your </span><span class='notice'>Apotheosis</span><span class='warning'> anymore!</span>")
+		new /obj/effect/temp_visual/plasma_soul(Dturf)//doesn't beam to you, so it just hangs around and then explodes instead of storing it inside of you.
 	log_combat(A, D, "gibbed (Plasma Fist)")
 	return
 
+/datum/martial_art/plasma_fist/proc/Apotheosis(mob/living/carbon/human/A, mob/living/carbon/human/D)
+	A.say("APOTHEOSIS!!", forced="plasma fist")
+	var/turf/boomspot = get_turf(A)
+	playsound(boomspot, 'sound/weapons/punch1.ogg', 50, TRUE, -1)
+	explosion(boomspot,plasma_power,plasma_power*2,plasma_power*4,ignorecap = TRUE)
```

Comment:
I told him it was probably fine to ignore bomb cap since the plasma_power is capped and this is admin grantable stuff. Its currently capped to 12, which requires 11 kills and your own suicide. It might be too high, thoughts?

Reply:
willing to change if people speak up

Reply:
whats the max explosion radius it can reach?

Reply:
At 12 its a 12/24/48. Maxcap is 5/10/20

Reply:
So a little over two max-caps.  That's as much as a traitor can buy on a normal shift.

B a l a n c e d

Reply:
you heard it here first. folks.

Reply:
Yeah, this is a fine reward for an 11 person killstreak.

Reply:
Someone forgot how square cube law works

Reply:
fludd shhh shshshsh hsh shut up! shut uppP!

Reply:
Guess this is fine then